src/react-app-env.d.ts

```diff
@@ -1 +1,45 @@
 /// <reference types="react-scripts" />
+
+export interface PostsFromServer {
```

Comment:
It's type for one post
```suggestion
export interface PostFromServer {
```